app/vue/src/client/preview/render.js

```diff
@@ -20,6 +20,10 @@ export default function render({
   Vue.config.errorHandler = showException;
 
   const component = story();
+  
+  if (typeof component === 'string') {
+    component = { template: component };
```

Comment:
component is a `const`, you can't reassign it, can you please fix it ? 
I think we need an example story as well.

Reply:
updated